### ISSUE TITLE
Unblock logic from freezing

### ISSUE THREAD

**Issue by ParadiseFallen** [stride3d/stride#1485]
**Is your feature request related to a problem? Please describe.**
When you drag your window or resize it game logic stop processing. Its not good. I can live with blocking rendering. But systems must work when you drag window. 

It is a bug.

It made networking much harder bc you will loose messages each time when drag. it will cause disconnections / desyncs.
Also it's good practice when your game logic running permanently

**Describe the solution you'd like**
Run game logic independently.

**Additional context**
After researching i found how to fix that issue.
Example for SDL:
https://pastebin.com/FGqiXhj2

Here are some questions that can aid in the description of the feature request:
> What are the usecases? 
 - networking
> What kind of options/settings are expected/wanted? 
- i can update source code for both sdl and winforms to use that
> What would the (pseudo) code look like? 
- pastebin. actual code for SDL. https://pastebin.com/FGqiXhj2
> Is the feature affecting behaviour in the editor? 
- not editor related
> Is there a component or datastructure? 
- engine lifecycle
> what kind of documentation is needed or needs to be updated?
- nothing. just mention it in release notes
> Should there be an example project?
- no

This is also solution for #709

**Comment by tebjan** [comment]
> * pastebin. actual code for SDL. https://pastebin.com/FGqiXhj2

Could you open a pull request so we can see a proper diff of the changes in the file?

**Comment by ParadiseFallen** [comment]
@tebjan Yes. i will do it tommorow (today but not now). Im tired for a long day and thats all what i can provide for now. Ull get PR tommorow

**Comment by ParadiseFallen** [comment]
@tebjan https://github.com/ParadiseFallen/stride/blob/e7720bf3752995c18aa3f4ef5ae57256ca50317b/sources/engine/Stride.Games/SDL/SDLMessageLoop.cs#L196

**Comment by tebjan** [comment]
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

**Comment by ParadiseFallen** [comment]
https://github.com/stride3d/stride/pull/1486 @tebjan

**Comment by ParadiseFallen** [comment]
conversation will goes into https://github.com/stride3d/stride/issues/1488
